commcare_connect/templates/account/password_reset_done.html

```diff
@@ -1,16 +1,14 @@
-{% extends "account/base.html" %}
+{% extends "account/base.tailwind.html" %}
 
+{% load static %}
 {% load i18n %}
-{% load account %}
+{% load crispy_forms_tags %}
 
 {% block head_title %}{% translate "Password Reset" %}{% endblock %}
 
 {% block inner %}
-    <h1 class="mb-4">{% translate "Password Reset" %}</h1>
-
-    {% if user.is_authenticated %}
```

Comment:
why is this no longer necessary? And if it is no longer necessary can we delete the old template?

Reply:
the template that was included here is a part of the `allauth` libraries templates.